subtrack.MAUI/Response/SubscriptionsMonthResponse.cs

```diff
@@ -0,0 +1,48 @@
+﻿using subtrack.MAUI.Services;
+using subtrack.DAL.Entities;
+using subtrack.MAUI.Data;
+
+namespace subtrack.MAUI.Response
+{
+    public class SubscriptionsMonthResponse
+    {
+        public DateTime CurrentDate { get; set; }
+        public IEnumerable<Subscription> Subscriptions { get; }
+        public IEnumerable<SubscriptionResponse> SubscriptionResponse { get; }
+
+        public SubscriptionsMonthResponse(DateTime dateTime)
+        {
+            CurrentDate = dateTime;
+        }
+
+        public IEnumerable<SubscriptionResponse> GetSubscriptionCostByMonth()
```

Comment:
This method can be removed. The view can call the method for getting the correct subs and calculating the cost itself and then another one to map them.